src/inline.tsx

```diff
@@ -8,14 +8,35 @@ import * as styles from './inline.css';
 import type {DataAttributes} from './utils/types';
 
 type Props = {
-    space: 0 | 2 | 4 | 8 | 12 | 16 | 24 | 32 | 40 | 48 | 56 | 64 | 'between' | 'around' | 'evenly';
+    space:
+        | -16
+        | -12
+        | -8
+        | -4
+        | -2
+        | 0
+        | 2
+        | 4
+        | 8
+        | 12
+        | 16
+        | 24
+        | 32
+        | 40
+        | 48
+        | 56
+        | 64
+        | 'between'
+        | 'around'
+        | 'evenly';
     alignItems?: 'flex-start' | 'flex-end' | 'center' | 'stretch' | 'baseline';
     children: React.ReactNode;
     className?: string;
     role?: string;
     'aria-labelledby'?: string;
     fullWidth?: boolean;
     dataAttributes?: DataAttributes;
+    wrap?: boolean;
```

Comment:
should the inline component wrap the content by default?

Reply:
I don't think so. And it would be a breaking change